lib/generate.js

```diff
@@ -135,6 +135,33 @@ generate.javascript.builtinPostprocessors = {
     "id": "id"
 }
 
+
+generate.es2015 = generate.es6 = function (parser, exportName) {
+    return "// Generated automatically by nearley\n" +
+        "// http://github.com/Hardmath123/nearley\n" +
+        "{\n" +
+        "const id = x => x[0];\n" +
+        parser.body.join('\n') +
+        "const grammar = {\n" +
+        "    Lexer: " + parser.config.lexer + ",\n" +
+        "    ParserRules: " + serializeRules(parser.rules, generate.es2015.builtinPostprocessors) + ",\n" +
+        "    ParserStart: " + JSON.stringify(parser.start) + "\n" +
+        "}\n" +
+        "if (typeof module !== 'undefined' && typeof module.exports !== 'undefined') {\n" +
+        "    module.exports = grammar;\n" +
+        "} else {\n" +
+        "    window." + exportName + " = grammar;\n" +
+        "}\n" +
+        "}\n";
+};
+generate.es2015.builtinPostprocessors = {
+    "joiner": "d => d.join('')",
+    "arrconcat": "d => [d[0]].concat(d[1])",
+    "arrpush": "d => d[0].concat([d[1]])",
```

Comment:
Could've done `([el, arr]) => [el].concat(arr)` and `([arr, el]) => arr.concat([el])`